amq/src/test/java/org/jboss/test/arquillian/ce/amq/AmqMultiReplicasPVTest.java

```diff
@@ -50,20 +50,18 @@
  * @author Ales Justin
  */
 @RunWith(Arquillian.class)
-// TODO -- change this once multi repl image is in prod
-@Template(url = "https://raw.githubusercontent.com/alesj/application-templates/amq1/amq/amq62-repl.json",
```

Comment:
Leave this template for now, until we get proper template in jboss-openshift repo.